README.md

```diff
@@ -152,6 +152,36 @@ preview = true
 
 Run `fortitude explain` to see which rules are in preview mode.
 
+### Excluding Files
```

Comment:
Maybe instead this should be in `docs/index.md`?

Actually, looking at ruff, they have some tools for extracting portions of the `README.md` into the site docs. We should look at that at some point

Reply:
For now, I've moved that section to the docs along with some other things from the README.